presto-docs/src/main/sphinx/release/release-0.269.rst

```diff
@@ -0,0 +1,37 @@
+=============
+Release 0.269
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add :func:`uuid` to return a unique identifier.
+* Add JDBC URL parameter doc for MySQL.
+
+Presto on Spark Changes
+_______________________
+* Add a new configuration property ``spark.memory-revoking-target`` to specify the percent to which memory pool should be occupied after revoke is completed.  This can be overridden by ``spark_memory_revoking_target`` session property.
+
+Accumulo Connector Changes
+__________________________
+* Replace :func:`uuid` from being connector specific to a standard function. In the process the return type has been changed from ``VARCHAR`` to ``UUID`` and existing use cases might need to perform a cast - ``CAST(UUID() AS VARCHAR)``.
+
+Delta Table Connector Changes
+_____________________________
+* New connector for reading Delta Lake tables(https://delta.io/) natively in Presto.
```

Comment:
New -> Add new.  Also, would be nice to link to the documentation for the connector